kml/type_utils.cpp

```diff
@@ -67,9 +67,9 @@ std::string GetLocalizedFeatureType(std::vector<uint32_t> const & types)
   return platform::GetLocalizedTypeName(c.GetReadableObjectName(type));
 }
 
-std::string GetPreferredBookmarkName(BookmarkData const & bmData, std::string const & languageOrig)
+std::string GetPreferredBookmarkName(BookmarkData const & bmData, std::string_view languageOrig)
 {
-  auto const languageNorm = languages::Normalize(languageOrig);
+  auto const languageNorm = languages::Normalize(std::string{languageOrig});
```

Comment:
So if we are moving in that way, why not modify languages::Normalize ?

Reply:
Return type should be string anyway according to use cases, but param can be changed, ok.

Reply:
So you don't need to create std::string here now.